zenodo/modules/metrics/api.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2016, 2017 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Zenodo Metrics API."""
+
+from __future__ import absolute_import
+
+import ipdb
+from flask import current_app
+from invenio_accounts.models import User
+from invenio_communities.models import Community
+from invenio_files_rest.models import FileInstance
+from werkzeug.local import LocalProxy
+
+from zenodo.modules.metrics.ext import ZenodoMetrics
+
+
+class ZenodoMetric(object):
+    """API class for Zenodo Metrics."""
+
+    # ipdb.set_trace() # ipdb.set_trace()
+
+    @staticmethod
+    def get_users():
+        """Get number of unique zenodo users.
+        """
+        return 1
+
+    @staticmethod
+    def get_visitors():
+        """Get number of unique zenodo users.
+        """
+        return 1
+
+    @staticmethod
+    def get_researchers():
+        """Get number of unique zenodo users.
+        """
+        return User.query.filter(
+            User.confirmed_at >= ZenodoMetrics().metrics_start_date(),
```

Comment:
```suggestion
            User.confirmed_at >= current_metrics.metrics_start_date,
```

- This would use the already configured/initialized extension (vs instantiating a new one)
- Since `metrics_start_date` is a property, there's no need to make a function call